### PR TITLE
feat(communities): re-implement community link unfurling

### LINKED ISSUE #4788: Community link is not resolved into invitation
### Description

1. create a community
2. copy the share community link
3. paste it in any chat on desktop

**Expected result:** community link is resolved into bubble with View / Join button (depends on the community type)

**Actual result:** the link is not resolved

Screenshot 2022-02-11 at 12 34 09

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #4788

Loads the community data and shows the data once loaded.

There is one known issue that I'll try to fix before we merge. The issue seems to be on status-go. The call to `requestCommunityInfo` never returns, so the thread I call it from never stops. I found that calling that was slow and froze the app until I called it in a thread, but if the call itself is just bugged, maybe I can remove the use of thread.

### CODE REVIEW

**Comment by jrainville** [comment]
The problem is fixed. You can review and test.

Don't merge until this status-go PR is merged tough: https://github.com/status-im/status-go/pull/2537

**Review by abhi-status — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Pull request merged**
